pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.wdl

```diff
@@ -100,11 +135,13 @@ workflow MultiSampleSmartSeq2SingleNucleus {
   }
 
 
+
+
   ### Pipeline output ###
   output {
```

Comment:
Can we add the pipeline version as a string output?

Reply:
This seems like a good idea